Changelog

```diff
@@ -1,9 +1,31 @@
 Changelog for Percona Toolkit
 
- * Improvement PT-1974: Support fingerprinting for --print in pt-kill (Thanks Iwo Panowicz)
- * Fixed bug   PT-1966: Fixed test (Thanks @yoku0825)
+v3.3.2 WIP release date not set yet
+
+ * Improvement PT-1417: Inconsistent creation of toolkit tables
```

Comment:
please see https://confluence.percona.com/display/DOC/Release+Notes+Template+and+Authoring+Guide 
This title needs to be about the Value we added, and it's hard to get it from there now.

Something like "Automated creation of tables in case of .... to avoid ... "